src/main/java/frc/robot/Robot.java

```diff
@@ -121,23 +187,29 @@ public void teleopPeriodic()
     }
     // barMotor.fsset(Math.pow(joystick.getRawAxis(4), 3));
     SmartDashboard.putNumber("bar enc", barMotor.getEncoder().getPosition());
+
+    // Emergency dump button in case the colour sensor senses the wrong colour
+    if (joystick.getRawButton(8)) {
+      ballIsBlue = false;
+      ballIsRed = false;
+      dumpBall = true;
```

Comment:
Is `dumpBall` just a placeholder for now and isn't being used?

Reply:
Yeah, it's a placeholder until we get the intake and turret finished